antlr_config/ManimLexer.g4

```diff
@@ -13,8 +13,12 @@ OPEN_GENERIC: '<' ;
 CLOSE_GENERIC: '>' ;
 OPEN_PARENTHESIS: '(' ;
 CLOSE_PARENTHESIS: ')' ;
+OPEN_CURLY_BRACKET: '{';
+CLOSE_CURLY_BRACKET: '}';
 CODE_COMMENT: '#' ~('\n' | '\r')* ('\n' | '\r')? -> skip;
 LET: 'let';
+DEF: 'def';
```

Comment:
This is quite minor but I personally prefer 'fun' or 'function' to 'def'. Will wait to see what everyone else thinks